pkg/kubelet/kubelet.go

```diff
@@ -1668,11 +1753,18 @@ func (kl *Kubelet) tryUpdateNodeStatus() error {
 	// cAdvisor locally, e.g. for test-cmd.sh, and in integration test.
 	info, err := kl.GetCachedMachineInfo()
 	if err != nil {
-		glog.Errorf("error getting machine info: %v", err)
+		// TODO(roberthbailey): This is required for test-cmd.sh to pass.
```

Comment:
Anyone can call the node api.  If cadvisor can't report anything, and the node already has something, should the node overwrite the capacity?  I would argue it should set zero if nothing is set, but if it's explicitly set (perhaps from the last time) it shouldn't be reset.  Not a massively strong feeling, but if cadvisor is dead I don't know that we should be making the node completely unschedulable (by forcing capacity to 0)